app/models/system_setting.rb

```diff
@@ -69,6 +69,7 @@ def peer_to_peer?
 #  confirmation_page_text_footer      :string
 #  confirmation_page_text_header      :string
 #  confirmation_page_text_link_header :string
+#  display_navbar                     :boolean          default(FALSE)
```

Comment:
Are these supposed to be checked in?

Reply:
Yes, we have a gem that automatically inserts these comments and keeps them up to date with new migrations.

Reply:
Thanks for the clarification @exbinary !